.github/workflows/deploy_to_aws.yml

```diff
@@ -44,10 +44,16 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
-      - name: Copy tfvars from S3 bucket for ${{ vars.ENV_NAME }}
+      - name: Checkout config repository
+        uses: actions/checkout@v4
+        with:
+          repository: 'opensupplyhub/ci-deployment'
+          path: 'terraform-config'
```

Comment:
can't see 'terraform-config' in 'ci-deployment'

Reply:
This is path in existing folder not path In the repository. I.e. it's the second argument in the `git clone` command:  `git clone  git://...res.git terraform-config`